Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -576,7 +576,20 @@ namespace Multiplayer
                     remoteAddress.c_str()) return;
             }
 
-            AZ_Printf("MultiplayerEditor", "Editor is listening for the editor-server...")
+            // Terminate any existing server launchers before launching a new one.
+            // It's possible for a rogue server launcher to exist if the Editor shutdown unexpectedly while running a previous multiplayer session.
+            // It's also common to open ServerLaunchers by hand for testing, but then to forget to shut it down before starting the editor play mode.
+            const AZStd::string serverExeFilename(AZ::Utils::GetProjectName() + ".ServerLauncher" + AZ_TRAIT_OS_EXECUTABLE_EXTENSION);
```

Comment:
This is making a big assumption that the filename of the ServerLauncher is renamed on disk.

Reply:
Not sure I follow. The ServerLauncher is typically <game>.ServerLauncher.exe.

Reply:
Not sure what we could do to prevent that. Is there a way to tag processes?

Reply:
No longer terminating based on name. I’ll warn users based on name and also tether the server lifetime to the editor.